server/tests/test_sse.py

```diff
@@ -32,3 +35,50 @@ async def test_events_listen_auth_then_not_allowed(
 # TODO: - test close on user revoked
 # TODO: - test close on backpressure (too many events pilling up)
 # TODO: - test bad accept type
+
+
+async def test_missed_events(minimalorg: MinimalorgRpcClients, backend: Backend) -> None:
+    from parsec._parsec import ActiveUsersLimit, authenticated_cmds
+
+    backend.config.sse_events_cache_size = 2
+    backend.events._last_events_cache = deque(maxlen=backend.config.sse_events_cache_size)  # pyright: ignore[reportPrivateUsage]
```

Comment:
This seems rather fragile, but I guess we don't have much choice (the alternative would be to avoid using `minimalorg` & `backend`, but this add too much complexity to the test...)